tensorflow/core/kernels/stateful_random_ops_gpu.cu.cc

```diff
@@ -75,10 +79,14 @@ void UpdateVariableAndFill_Philox<GPUDevice, Distribution>::operator()(
                                              FillKernel<Distribution>, 0, 0);
 
   int zero = 0;
+#if GOOGLE_CUDA  
   cudaMemcpyToSymbol(thread_counter, &zero, sizeof(int));
+#else // TENSORFLOW_USE_ROCM#
+  hipMemcpyToSymbol(HIP_SYMBOL(thread_counter), &zero, sizeof(int));
```

Comment:
hopefully this op would stay as-is for some time before moving to become fully dynamically loaded...